src/api/emails/send-application-submitted-emails/index.ts

```diff
@@ -13,44 +14,90 @@ import { SuccessResponse, ApplicationSubmissionEmailVariables, Application } fro
       companyName: company.companyName,
       requestedStartDate: formatDate(policyAndExport.requestedStartDate),
+    };
+
+    // email variables for sending email to application owner of application
+    const sendEmailVars = {
+      ...sharedEmailVars,
+      name: getFullNameString(owner),
+      emailAddress: email,
     } as ApplicationSubmissionEmailVariables;
 
-    // send "application submitted" email receipt to the applicant
-    const accountSubmittedResponse = await sendEmail.applicationSubmitted.account(sendEmailVars);
+    // email variables for sending email to business contact named on application
+    const sendContactEmailVars = {
+      ...sharedEmailVars,
+      name: getFullNameString(businessContactDetail),
+      emailAddress: businessContactDetail.email,
+    } as ApplicationSubmissionEmailVariables;
+
+    // checks if application owner email on application is the same as contact email provided
+    const isOwnerSameAsContact = isOwnerSameAsBusinessContact(email, businessContactDetail.email);
+
+    console.info('Sending application submitted email to application account owner: ', sendEmailVars.emailAddress);
+    // send "application submitted" email receipt to the applcation owner applicant
+    const accountSubmittedResponse = await sendEmail.application.submittedEmail(sendEmailVars);
 
     if (!accountSubmittedResponse.success) {
       throw new Error('Sending application submitted email to owner/account');
     }
 
+    /**
+     * if the contact email address is different to the application owner applicant
+     * then it sends the same "application submitted" email receipt to the contact email address
+     */
+    if (!isOwnerSameAsContact) {
+      console.info('Sending application submitted email to business contact email: ', sendContactEmailVars.emailAddress);
+      const contactSubmittedResponse = await sendEmail.application.submittedEmail(sendContactEmailVars);
+
+      if (!contactSubmittedResponse.success) {
+        throw new Error('Sending application submitted email to contact');
+      }
+    }
+
     // get email template IDs depending on the submitted answers
     const templateIds = getApplicationSubmittedEmailTemplateIds(application);
 
     // send "application submitted" email to the underwriting team
-    const underwritingTeamSubmittedResponse = await sendEmail.applicationSubmitted.underwritingTeam(sendEmailVars, csvPath, templateIds.underwritingTeam);
+    const underwritingTeamSubmittedResponse = await sendEmail.application.underwritingTeam(sendEmailVars, csvPath, templateIds.underwritingTeam);
 
     if (!underwritingTeamSubmittedResponse.success) {
       throw new Error('Sending application submitted email to underwriting team');
     }
 
     // send "documents" email to the applicant depending on submitted answers
     if (templateIds.account) {
+      console.info('Sending documents email to business account owner: ', sendEmailVars.emailAddress);
```

Comment:
`business contact` - will this always be the case? The comment above specifically mentions `applicant`

Reply:
changed to `application owner`